src/altimate.ts

```diff
@@ -291,6 +326,48 @@ export class AltimateRequest {
     }
   }
 
+  async downloadFileLocally(
+    url: string,
+    filePath = "manifest.json",
+  ): Promise<string> {
+    const tempFolder = os.tmpdir();
+
+    mkdir(tempFolder, { recursive: true }, (err) => {
+      console.log(err);
+    });
+
+    const destinationPath = join(tempFolder, filePath);
```

Comment:
This path might conflict if multiple projects are ran simultaneously. maybe better use a uuid?

Reply:
It automatically creates a uuid. It's not a fixed path

Reply:
I checked it, it is not a unique path. it is shared with other stuff.